library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/Section.java

```diff
@@ -39,11 +40,13 @@
      * @param itemResourceId layout resource for its items
      * @param loadingResourceId layout resource for its loading state
      * @param failedResourceId layout resource for its failed state
+     * @param emptyResourceId layout resource for its empty state
      */
-    public Section(int itemResourceId, int loadingResourceId, int failedResourceId) {
```

Comment:
@luizgrp Hi, I'm the fan of Joshua Bloch's Effective Java. I'm referring his Builder pattern - https://stackoverflow.com/a/1953567/72437

But, do you have any idea, how we can apply such technique, in `Section` class as `Section` is abstract class.